### DISCUSSION TITLE
不支持openai协议的商业接口如何接入dify

### DISCUSSION THREAD

**Discussion by yangxiaoshuai2333** [langgenius/dify#6822] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

通义法睿这个模型不支持openai协议，只能使用dashscope接口，这种模型如何接入dify呢。openai自定义界面无法接入，qwen界面没有这个模型选项，需要自己写yaml配置么，dify是通过docker部署的，如果是配置的话位置在哪里呢

**Comment by Cuihongsen** [comment]
使用 [one-api](https://github.com/songquanpeng/one-api) 自己转协议

**Comment by RRRoger** [comment]
这题我会 我用阿里pdf插件也有这个问题 这个要自己写一个中转的服务 去兼容openai的接口规则 然后用openllm那个provider去链接就可以

**Reply by yangxiaoshuai2333** [reply]
好吧，要么框架转openai要么自己写兼容接口
